torch_frame/config/image_embedder.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import Callable
+
+from PIL import Image
+from torch import Tensor
+
+
+class ImageEmbedder(ABC):
+    r"""Parent class for the :obj:`image_embedder` of
+    :class:`ImageEmbedderConfig`. You should implement the
+    :meth:`forward_embed` which takes a list of images and returns
+    embeddings tensor. If you have your own retrieve function to
+    retrieve images from the paths, please override :meth:`forward_retrieve`
+    which takes the paths to images and return a list of
+    :obj:`PIL.Image.Image`.
+    """
+    def __init__(self, *args, **kwargs):
+        pass
+
+    def forward_retrieve(self, path_to_images: list[str]) -> list[Image.Image]:
+        r"""Retrieval function that reads a list of images from
+        a list of file paths with the :obj:`RGB` mode.
+        """
```

Comment:
nit make """ without line change. Same for other places.

Reply:
The pre-commit will automatically make """ with line change.